.github/workflows/publish-bundle.yaml

```diff
@@ -0,0 +1,32 @@
+name: publish-bundle
+
+on:
+  workflow_call:
+    secrets:
+      CHARMCRAFT_AUTH:
+        required: true
+    inputs:
+      bundle_variant:
+        required: true
+        type: string
```

Comment:
How about we make this guy a `choice` instead of `string`? When more variants come, it would give a clear visibility of what's supported? I mean otherwise, you'll need to search for the supported options.

Reply:
Ah, that's a bummer.